src/main/resources/application.yml

```
@@ -1,5 +1,5 @@
 spring:
   profiles:
     group:
-      dev: dev, common
-      prod: prod, common
\ No newline at end of file
+      local: local, common
+      dev: dev, common
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)